.pre-commit-config.yaml

```diff
@@ -50,7 +50,7 @@ repos:
     - id: trailing-whitespace
 
 -   repo: https://gitlab.com/pycqa/flake8
-    rev: 3.7.9
+    rev: 3.9.2
```

Comment:
This was updated due to a bug in an earlier version that causes flake8 to error on `# type: ignore[flag]` comments